Workbooks/UsageMetrics/Usage - Users/Usage - Users.workbook

```diff
@@ -0,0 +1,710 @@
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components",
+        "visualization": "barchart"
+      },
+      "conditionalVisibilities": [
+        {
+          "parameterName": "SplitBy",
+          "comparison": "isNotEqualTo"
+        },
+        {
+          "parameterName": "ByType",
+          "comparison": "isNotEqualTo",
+          "value": "Time"
+        }
+      ],
+      "conditionalVisibility": {
+        "parameterName": "SplitBy",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "query - 9"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let timeRange = {TimeRange};\r\nlet threshold = 2000; // Apdex threshold in milliseconds\r\nunion {Whoused}\r\n    | where timestamp > ago(timeRange)\r\n    {Cohort}\r\n    | where iif('*' in ({UserEvent}), 1==1, name in ({UserEvent}))\r\n    | extend Apdex=case(\r\n    duration <= threshold, \"Satisfied\",\r\n    duration <= 4*threshold, \"Tolerating\",\r\n    \"Frustrated\")\r\n    | summarize SESSIONS = dcount(session_Id), EVENTS = count() , PERFORMANCE=\"Null\", TotalSatisfied=sumif(itemCount, Apdex == \"Satisfied\"), TotalTolerating=sumif(itemCount, Apdex == \"Tolerating\"), TotalFrustrated=sumif(itemCount, Apdex == \"Frustrated\")\r\n    | project SESSIONS, EVENTS, ApdexScore = (TotalSatisfied + (TotalTolerating / 2)) / todouble(TotalSatisfied + TotalTolerating + TotalFrustrated)\r\n    | extend PERFORMANCE = case(\r\n    ApdexScore <= 0.5, \"Unacceptable\",\r\n    ApdexScore <= 0.7, \"Poor\",\r\n    ApdexScore <= 0.85, \"Fair\",\r\n    ApdexScore <= 0.94, \"Good\",\r\n    \"Excellent\")\r\n    | project SESSIONS, EVENTS, PERFORMANCE\r\n    | evaluate narrow()\r\n    | project Column, Value, id = case(Column =='SESSIONS', 'community-Workbooks%2FUsageMetrics%2FUsage%20-%20Sessions', Column =='EVENTS', 'community-Workbooks%2FUsageMetrics%2FUsage%20-%20Events','Community-Workbooks%2FPerformance%2FApdex'), Note = case(Column =='PERFORMANCE', 'Threshold 2000 milliseconds', '­\t')",
```

Comment:
for places returning counts, i'd set the number formatting to know that it is counts so it can be "nificied":
![image](https://user-images.githubusercontent.com/10158007/72944709-97a22f80-3d2e-11ea-991a-8d935218c609.png)
![image](https://user-images.githubusercontent.com/10158007/72944722-9ffa6a80-3d2e-11ea-93f1-85d8666603f5.png)

Reply:
this comment applies generally throughout all the grids showing counts, etc.  for the ones showing bars of counts, you also should set the minimum value to 0, otherwise it calculates bars between the largest and smallest value

Reply:
is there a way to change font size? 
![image](https://user-images.githubusercontent.com/48222200/73102218-3e5d0c00-3ea6-11ea-87a8-6e998c71348d.png)

Reply:
you have the renderer set to bignumber, so that field will always be big.  you could leave the number there for the duration or whatever, and put that unnacceptable text into another column, and use that as the subtitle instead of the left part of the tile?

Reply:
i see you did this generally in tiles, but not in any grids?  they're all showing unformatted numbers?

![image](https://user-images.githubusercontent.com/10158007/73683158-9f3ece00-4676-11ea-8e52-e5645ae46a37.png)

![image](https://user-images.githubusercontent.com/10158007/73683171-a534af00-4676-11ea-8b30-dbf5fd94c9ae.png)